src/kaiteki/assets/instances.json

```diff
@@ -288,16 +288,19 @@
     "shortDescription": "Fun hangout for furries and normies alike.",
     "favicon": "https://stop.voring.me/assets/about-icon.86c5c1fe.png",
     "rules": [
-      "None of the following:",
+      "Please read the whole TOS for the full rules list 🙇",

```

Comment:
The whole list of rules should be included, without clarification description.
The design of the instance page puts emphasis on reading the rules, before being offered the login option.